Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/ViewModels/EncryptFileViewModel.cs

```diff
@@ -75,18 +66,33 @@ public EncryptFileViewModel(IDesignServices services) : base(services)
         /// </summary>
         public DesignInArgument<bool> ContinueOnError { get; set; } = new DesignInArgument<bool>();
 
+        /// <summary>
+        /// The file that you want to encrypt.
+        /// </summary>
+        public DesignInArgument<IResource> InputFile { get; set; } = new DesignInArgument<IResource>();
+
+        /// <summary>
+        /// The file that you want to encrypt.
+        /// </summary>
+        public DesignOutArgument<CryptographyRemoteItem> EncryptedFile { get; set; } = new DesignOutArgument<CryptographyRemoteItem>();
```

Comment:
See comments left for the decript view model.

Reply:
resolved